src/daemon/custom_image_host.cpp

```diff
@@ -49,6 +49,56 @@ auto multipass_default_aliases(mp::URLDownloader* url_downloader)
 
     default_aliases.insert({"core", core_image_info});
 
+    // Image information on 16.04
+    const QString url_16{"https://cloud-images.ubuntu.com/releases/16.04/release/ubuntu-16.04-server-cloudimg-amd64-disk1.img"};
+    const auto last_modified = url_downloader_16->last_modified({url_16});
+    const auto sha256_sums_16 =
+        url_downloader_16->download({"https://cloud-images.ubuntu.com/releases/16.04/release/SHA256SUMS"}).split('\n');
+    QString hash_16;
+
+    for (const auto& line : sha256_sums_16)
+    {
+        if (line.endsWith("ubuntu-16.04-server-cloudimg-amd64-disk1.img"))
+        {
+            hash = QString(line.split(' ').first());
+            break;
+        }
+    }
+
+    // Image information on 18.04
+    const QString url_18{"https://cloud-images.ubuntu.com/releases/18.04/release/ubuntu-18.04-server-cloudimg-amd64.img"};
+    const auto last_modified = url_downloader_18->last_modified({url_18});
```

Comment:
Same as above, `last_modified` has already been declared and `url_downloader_18` has not been declared:)